Instagram Liker Bot/Instagram_Liker_Bot.py

```diff
@@ -0,0 +1,44 @@
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.common.keys import Keys
+from getpass import getpass
```

Comment:
What is this getpass?

Reply:
It's for keeping the password encrypted

Reply:
So that if the password is entered publicly it stays encrypted